lib/matplotlib/backend_tools.py

```diff
@@ -138,6 +138,14 @@ class ToolToggleBase(ToolBase):
     Toggleable tool
 
     Every time it is triggered, it switches between enable and disable
+
+    Parameters
+    ----------
+    *args
+        Variable length argument to be used by the Tool
+    **kwargs
```

Comment:
What is the standard to document optional kwargs? like `toggled` in this particular case?